Automattic/wp-calypso
# Adding Facebook Messenger card to Marketing Tools

This PR adds a Facebook Messenger card to Calypso's Marketing Tools page. The card itself promotes the [Facebook Chat plugin](https://wordpress.com/plugins/facebook-messenger-customer-chat). More context about this effort is at pbBQWj-vh-p2.

Clicking the button takes fires a tracks event (`calypso_marketing_tools_facebook_messenger_button_click`) and takes the user to the Facebook Chat plugin.

The plugin description doesn't appear to be translated so the card on the Marketing page is currently showing only to EN users. The card strings will be translated anyways, so they're ready to launch when the plugin copy is translated.

Here's what the card should look like:
<img width="1065" alt="Screen Shot 2020-11-03 at 7 25 39 AM" src="https://user-images.githubusercontent.com/35781181/97986535-da0d9800-1da7-11eb-82fc-c78059f09c4e.png">

And here's a screenshot of the Tracks event firing after clicking the button:
<img width="861" alt="Screen Shot 2020-11-03 at 7 25 24 AM" src="https://user-images.githubusercontent.com/35781181/97986576-eabe0e00-1da7-11eb-945d-4f6ea2ea7fa7.png">


#### Testing instructions
* Checkout this branch and start Calypso
* Navigate to /marketing/tools/[siteslug] with an EN user
* Confirm that the card show in the first row, second column.
* Click the button and confirm that clicking the button takes you to the plugins page (in a new window)
